### ISSUE TITLE
Xeno Tunnels should be more visible in hive vision

### ISSUE THREAD

**Issue by Vermidia** [RMC-14/RMC-14#7333]
## Description

You can't really see them in hivemind vision anymore.

**Reproduction**

**Screenshots**

**Additional context**

**Comment by noctyrnal** [comment]
i am very certain that compared to the original sprites they are brighter (OG sprites were pure black)

**Comment by noctyrnal** [comment]
<img width="32" height="32" alt="Image" src="https://github.com/user-attachments/assets/8ef2d854-e7ba-42ae-9212-4869ac3765a6" /> vs <img width="32" height="32" alt="Image" src="https://github.com/user-attachments/assets/a15f7c6d-f052-41ad-a0b0-2fd693088b99" />

**Comment by Vermidia** [comment]
they had a weird outline which made them visible

**Comment by Vermidia** [comment]
Fixed via sprite change
